modules/mailchimp_lists/includes/mailchimp_lists.admin.inc

```diff
@@ -10,6 +10,10 @@
  */
 function mailchimp_lists_overview_page() {
   $mc_lists = mailchimp_get_lists();
+  $account = mailchimp_api_account();
+  $output = '<h2>' . $account->account_name . '</h2>';
```

Comment:
This header is also added by `mailchimp_account_select_form()` so creates a double header on the lists page. Could this line be removed?